ecosystem/sep-0031.md

```diff
@@ -34,20 +34,14 @@ NigeriaPay will utilize its European rail, enabled with EuroPay Anchor service,
 
 ## Authentication
 
-Anchors should support [SEP-10](sep-0010.md) web authentication to ensure the counterparty they're interoperating with is actually who they say they are.  Clients must submit the JWT obtained via the SEP-10 authentication flow to all API endpoints.
+Anchors should support [SEP-10](sep-0010.md) web authentication to ensure the counterparty they're interoperating with is actually who they say they are.  Clients must submit the JWT obtained via the SEP-10 authentication flow to all API endpoints except `/info`.
 
-The JWT should be included in all requests as request header:
+The JWT should be included as a request header:
 ```
 Authorization: Bearer <JWT>
 ```
 
-Any API request the fails to meet proper authentication should return a 401 Unauthorized response with the error payload:
-
-```
-{
-  "type": "authentication_required"
-}
-```
```

Comment:
We use this response in SEP-6 because there are other responses defined that share the same status code. For example, both SEP-6 bad auth and customer_info_needed responses use 403 and include the `type` attribute to disambiguate.

In SEP-31, the 401 status code is only used for bad auth, so there is no need to specify a `type`.